Automattic/wp-calypso
# Patterns Assembler: Create blank canvas cta

#### Proposed Changes

- Create a Blank Canvas CTA in UDP
- Create Build your own Layout step in stepper flows

#### Testing Instructions

- Visit http://calypso.localhost:3000/setup/designSetup?siteSlug={site_slug}&flags=signup/design-picker-pattern-assembler
- Confirm that the Blank Canvas CTA is in in the UDP
- Click on Get Started button should navigate to new step 'buildYourOwnLayout'
- 'buildYourOwnLayout' step is empty by far, click Back button should go back to DesignSetup step

https://user-images.githubusercontent.com/10071857/186340773-e59c1dc2-2273-4e72-aee0-dd06e90b1de7.mp4

## Reference

Related to #66785
